lib/utils.pm

```diff
@@ -1078,7 +1094,25 @@ sub set_hostname {
     assert_script_run "uname -n|grep $hostname";
     systemctl 'status network.service';
     save_screenshot;
-    assert_script_run "if systemctl -q is-active network.service; then systemctl reload-or-restart network.service; fi";
+
+    if (systemctl('is-active NetworkManager', ignore_failure => 1) == 0) {
+        # NetworkManager appears to do nothing when network.service is restarted on TW 20230514
```

Comment:
is it bug or feature?

Reply:
I have created a bug to check this: https://bugzilla.opensuse.org/show_bug.cgi?id=1211677